analytics/analytics/views.py

```diff
@@ -0,0 +1,18 @@
+from typing import Any
+from django.http import HttpRequest, HttpResponse
+import json
+
+import sentry_sdk
+
+from analytics.job_log_uploader import upload_job_log
+
+
+def webhook_handler(request: HttpRequest) -> HttpResponse:
+    job_input_data: dict[str, Any] = json.loads(request.body)
+
+    if job_input_data.get("object_kind") != "build":
+        raise ValueError("Not a build event")
```

Comment:
This code is copied from other webhooks, but AFAICT it's there so that it doesn't attempt to process events that aren't related to jobs.

Reply:
The main reason I ask is because if we don't want this to be treated as an error to be dealt with (and so we don't get this error in sentry), we could return a 400 here instead of raising an exception.

Reply:
I changed it to return a 400 but also catch a sentry error, since I think a misconfiguration of our webhooks is indicative of a larger problem.